builder/llpcBuilder.h

```diff
@@ -131,6 +131,25 @@ inline static void InitializeBuilderPasses(
 class Builder : public llvm::IRBuilder<>
 {
 public:
+    // The group operations the builder can consume.
+    // NOTE: We rely on casting this implicitly to an integer, so we cannot use an enum class.
+    enum GroupOperation
```

Comment:
please change to enum GroupOperation : uint32_t to reduce the difference between different compilers

Reply:
The comment above explains why I can't `NOTE: We rely on casting this implicitly to an integer, so we cannot use an enum class`

Reply:
I think class enum GroupOperation : uint32_t will block you implicitly convert, but enum GroupOperation : uint32_t still support casting this implicitly to an integer, please have a try. thanks!